### ISSUE TITLE
Commas in results data?

### ISSUE THREAD

**Issue by stevieflow** [devinit/D-Portal#640]
Hello

We were previewing some bigger target and actual numbers in IATI results:

![d-preview (12)](https://user-images.githubusercontent.com/1209944/156619164-86348f63-a6cc-4f3c-8e6b-f86544adc431.png)

They are difficult to parse, by sight - could it be possible to include the relevant commas in the numbers (as you do for budgets and transaction values)?

Thanks!

**Comment by stevieflow** [comment]
Thanks @notshi 

Actually - ignore me!

I noticed that the schema declares this value field as a string: https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/result/indicator/period/target/

And - on checking:

https://datasette.codeforiati.org/iati/result_indicator_period_actual?value__notblank=1&_sort_desc=value

There's a lot of strings in this field!

I can therefore format the data with commas.  Case closed.  Thanks for your advice

**Comment by notshi** [comment]
> I can therefore format the data with commas.

Publishing with commas could make it hard to discern that we are dealing with numbers (as a data consumer).
This will also affect the completion percentages you find in the result section of SAVi.
Also in different locales, commas can mean decimal points which can make the numbers even more confusing.

**Comment by stevieflow** [comment]
Ah ha!  Thanks @notshi - Ill revert back to "numbers" therefore 

To the original query - given the widespread mix of strings and numbers, is there anything you can do at the d-portal end to format the numbers?

**Comment by notshi** [comment]
Yup, we will take a look and see how best we can do this. Should we re-open this issue?
